src/Interactive/DesktopHost/InteractiveHostEntryPoint.cs

```diff
@@ -15,7 +15,7 @@ private static int Main(string[] args)
 
             try
             {
-                InteractiveHost.Service.RunServer(args);
+                InteractiveHost.Service.RunServerAsync(args);
```

Comment:
>RunServerAsync [](start = 40, length = 14)

This is good, but we need to await this async method. C# allows you to make Main async as well (returning `Task<int>`).